src/modules/services/main.py

```diff
@@ -35,37 +38,64 @@ def run():
-                return "Cannot enable systemd service {}".format(svc['name']), \
-                       "systemctl enable call in chroot returned error code {}".format(ec)
+                return ("Cannot enable systemd service {}".format(svc['name']),
+                        "systemctl enable call in chroot returned error code",
+                        "{}".format(ec)
+                        )
             else:
-                libcalamares.utils.debug("Cannot enable systemd service {}".format(svc['name']))
-                libcalamares.utils.debug("systemctl enable call in chroot returned error code {}".format(ec))
+                libcalamares.utils.debug(
+                    "Cannot enable systemd service {}".format(svc['name'])
+                    )
+                libcalamares.utils.debug(
+                    "systemctl enable call in chroot returned error code",
+                    "{}".format(ec)
+                    )
 
     # enable targets
     for tgt in targets:
-        ec = libcalamares.utils.target_env_call(['systemctl', 'enable', '{}.target'.format(tgt['name'])])
+        ec = libcalamares.utils.target_env_call(
+            ['systemctl', 'enable', '{}.target'.format(tgt['name'])]
+            )
 
         if ec != 0:
             if tgt['mandatory']:
-                return "Cannot enable systemd target {}".format(tgt['name']), \
-                       "systemctl enable call in chroot returned error code {}".format(ec)
+                return ("Cannot enable systemd target {}".format(tgt['name']),
+                        "systemctl enable call in chroot returned error code",
+                        "{}".format(ec)
+                        )
             else:
-                libcalamares.utils.debug("Cannot enable systemd target {}".format(tgt['name']))
-                libcalamares.utils.debug("systemctl enable call in chroot returned error code {}".format(ec))
+                libcalamares.utils.debug(
+                    "Cannot enable systemd target {}".format(tgt['name'])
+                    )
+                libcalamares.utils.debug(
+                    "systemctl enable call in chroot returned error code",
+                    "{}".format(ec)
+                    )
 
     for dbl in disable:
-        ec = libcalamares.utils.target_env_call(['systemctl', 'disable', '{}.service'.format(dbl['name'])])
+        ec = libcalamares.utils.target_env_call(
+            ['systemctl', 'disable', '{}.service'.format(dbl['name'])]
+            )
 
         if ec != 0:
             if dbl['mandatory']:
-                return "Cannot disable systemd service {}".format(dbl['name']), \
-                       "systemctl disable call in chroot returned error code {}".format(ec)
+                return ("Cannot disable systemd service",
+                        "{}".format(dbl['name']),
+                        "systemctl disable call in chroot returned error code",
+                        "{}".format(ec))
```

Comment:
for what reason ever: "{}".format(ec) fails when stand alone - had to investigate